### ISSUE TITLE
The CSS components on versions > 5.0.0 are broken

### ISSUE THREAD

**Issue by is-sam** [Shopify/polaris-react-archive#4750]
## Issue summary

When I use the `styles.css` from Polaris version **5.0.0**, as in [this example](https://github.com/Shopify/polaris-react/tree/main/examples/cdn-styles), the CSS components work but obviously with the old Shopify styles. When I switch to the latest version **7.3.1** as in [the documentation ](https://github.com/Shopify/polaris-react#using-the-css-components) the components don't work correctly.

## Expected behavior

The components should render correctly as in the [Polaris documentation](https://polaris.shopify.com/components/actions/button#navigation).

## Actual behavior

The components are broken and don't show up correctly. 

## Steps to reproduce the problem
Here is an example of a button using Polaris styles.css version **5.0.0** vs **7.3.1**:

Let's use this Polaris component example present in the documentation:
```html
  <div>
      <button class="Polaris-Button Polaris-Button--primary" type="button">
      <span class="Polaris-Button__Content">
          <span class="Polaris-Button__Text">Save theme</span>
      </span>
      </button>
      <div id="PolarisPortalsContainer"></div>
  </div>
```

Importing the CSS file in the `<head>` of my HTML:

### Polaris@5.0.0
```html
    <link rel="stylesheet" href="https://unpkg.com/@shopify/polaris@5.0.0/dist/styles.css"/>
 ```

**Result:**
![image](https://user-images.githubusercontent.com/15368990/143926905-24378b6e-3878-402b-a483-9e693b2a9b4e.png)

### Polaris@7.3.1
```html
    <link rel="stylesheet" href="https://unpkg.com/@shopify/polaris@7.3.1/build/esm/styles.css"/>
 ```

**Result:**
![image](https://user-images.githubusercontent.com/15368990/143927055-374e9e3d-3a6c-4a6b-a219-dbb5efd9e864.png)

## Reduced test case

Here is an example on [CodeSandbox](https://codesandbox.io/s/empty-mountain-rw8b4).

## Specifications

- Are you using the React components? (Y/N): No
- Polaris version number: 7.3.1
- Browser: Chrome
- Device: PC
- Operating System: Windows 10

**Comment by chenster** [comment]
Still broken March 16, 2022

**Comment by schmoove** [comment]
Still broken on 9.2.2 – the instructions in the docs don't work.

**Comment by martenbjork** [comment]
See fix in #6650. If this bug resurfaces, please let us know! 🙏
